src/app.ts

```diff
@@ -1,5 +1,5 @@
-const express = require("express");
-const jwt = require("jsonwebtoken");
+import express = require("express");
```

Comment:
*Old way:*
`const express = require("express");`
*New way:*
`import express from 'express'`

sometimes the library is not prepared for it, so we have to make a workaround:
`import * as jwt from 'jsonwebtoken'`